umh-core/pkg/diagnosis/declare.go

```
@@ -0,0 +1,130 @@
+	External bool
+}
+
+// Capable returns the instruments the environment satisfies, in declared order.
+func (s Signal[S]) Capable(env Environment) []Instrument[S] {
+	capable := make([]Instrument[S], 0, len(s.Instruments))
+	for _, inst := range s.Instruments {
+		satisfied := true
+		for _, req := range inst.Requires {
+			if !env.Has(req) {
+				satisfied = false
+				break
```

golangci-lint (2.12.2)

[error] break with no blank line before

(nlreturn)